src/status_im/ui/screens/popover/views.cljs

```diff
@@ -34,7 +35,8 @@
         current-popover   (reagent/atom nil)
         update?           (reagent/atom nil)]
     (reagent/create-class
-     {:component-will-update (fn [_ [_ popover _]]
+     {:component-did-mount   (fn [] (reset! current-popover nil))
```

Comment:
could you elaborate?

Reply:
I was trying to improve dev experience and experimenting.. this was not the problem.. will remove

Reply:
yeah better to do not change production code for dev experience. if you want to have a better experience with popover you could just temporary comment atom and use popover object, in that case, it will work without animation but will work even after refresh

Reply:
understood 
removed
will keep it in mind about it next time